haskell/1/index.md

```diff
@@ -183,14 +129,14 @@ executable bassbull
   default-language:    Haskell2010
 ```
 
-A few notable changes:
+A few things to note:
 
-* Set the description so Cabal would stop squawking about it.
-* Set `hs-source-dirs` to `src` so Cabal knows where my modules are.
-* Added a named executable stanza to the Cabal file so I can build a binary by that name and run it.
-* Set `main-is` to `Main.hs` in the executable stanza so the compiler knows what main function to use for that binary.
-* Set `ghc-options` to `-Wall` so we get the *rather* handy warnings GHC offers on top of the usual type checking.
-* Added the libraries our project will use to `build-depends`.
+* The description tells people what the package is about.
+* The `hs-source-dirs` includes `src` so Cabal knows where my modules are.
```

Comment:
Is it worth changing Cabal to Stack here to avoid confusion? It's definitely _true_ that Cabal needs to know these things, but I wonder if introducing both concepts is confusing. That said, the presence of a .cabal file is probably enough to introduce that confusion.